testnet/utils.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func copy(src, dst string) (int64, error) {
```

Comment:
Surprising this doesn't conflict with the `copy` builtin.

Reply:
I just changed it, to avoid confusion